tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml

```diff
@@ -1,6 +1,6 @@
 jobs:
 - job: 'build'
-  pool: 'onnxruntime-tensorrt8-winbuild-T4'
+  pool: 'onnxruntime-Win2022-GPU-T4'
```

Comment:
Let's keep this file unchanged. We may do it later. The "onnxruntime-tensorrt8-winbuild-T4" belongs to the TRT team. It may have special settings that I'm not aware of.